modules/postgres/README.md

```diff
@@ -20,6 +20,20 @@ To find out more about these metrics and why they are important to monitor, read
 - User with granted `pg_monitor`
   or `pg_read_all_stat` [built-in role](https://www.postgresql.org/docs/current/predefined-roles.html).
 
+Without additional configuration Netdata will attempt to use the default postgres user - but a separate netdata user can be created for this purpose. If you have PostgreSQL 10+ and want to use Netdata to monitor statistics normally reserved for superusers grant the netdata user pg_monitor permissions. Some of the advanced metrics also require additional permissions as mentioned in [metrics](https://github.com/netdata/go.d.plugin/tree/master/modules/postgres#metrics).
+
+1. Login to a psql session as a user with CREATEROLE priviliges
+2. `postgres=# create user netdata with password '<PASSWORD>';`
```

Comment:
If we suggest passwordless we can add data collection jobs with netdata user, so it would be: create the user, restart netdata service.

Reply:
In addition, the can be not a list but a code block a user can copy/paste. See https://github.com/netdata/go.d.plugin/tree/master/modules/mysql#requirements

Reply:
How about now?